src/app/modules/core/services/beacon-node.service.ts

```diff
@@ -11,9 +11,20 @@ import {
   NodeConnectionResponse,
 } from 'src/app/proto/validator/accounts/v2/web_api';
 
+import {
+  ValidatorBalances,
+} from 'src/app/proto/eth/v1alpha1/beacon_chain';
+
 const POLLING_INTERVAL = 3000;
 const BEACON_API_SUFFIX = '/eth/v1alpha1';
 
+export class BeaconState {
```

Comment:
Beacon state is confusing naming because it contradicts with the [`BeaconState`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/beacon-chain.md#beaconstate) data structure

Reply:
Perhaps we should use something like BeaconNodeState ?

Reply:
sure i will rename it